code/modules/cargo/randomstock.dm

```diff
@@ -74,8 +74,6 @@ var/list/global/random_stock_common = list(
 	"light" = 1.8,
 	"aid" = 4,
 	"flame" = 2,
-	"crayons" = 1.5,
```

Comment:
Against removing those things from cargo, not everything must have an use, as plushies and toys have some rp value, mostly as gifts.

Reply:
You can get toys from arcade machines. I've rarely seen toys or action figures used by people when I put it in the bar.

Reply:
My point still stands, a lot of the things in the warehouse are flavor and not just used as gear or something, and I am sure that you can find some rares bags in the warehouse, undo those changes.

Reply:
I rather not.

Reply:
I am inclined to agree with Alberyk. The warehouse is not meant to be an assortment of purely useful items, it is meant to simply be an assortment of random items that vary from, yes useless, to useful. Trending away from this paves a way to worse implications.

Reply:
We'll see what the maintainers have to say. I've played cargo for 2 months now and I've never seen an actual use for any of the junk I've removed from the warehouse, other than "Oh this is trash."

Reply:
As I pointed out, some bags can either only be found in hand of antags or in the warehouse, and I had a different experience when it comes down to plush toys in the warehouse, because I do seek them to either give them out as a gift or just keep them.

Reply:
I'll re add them, then. It should justify the 20 loot increase.